src/storage/store/column_chunk.cpp

```diff
@@ -403,67 +403,59 @@ bool ColumnChunk::sanityCheck() {
 
 ColumnChunkMetadata ColumnChunk::getMetadataToFlush() const {
     KU_ASSERT(numValues <= capacity);
-    StorageValue minValue = {}, maxValue = {};
+    std::optional<StorageValue> minValue, maxValue;
     if (capacity > 0) {
+        std::optional<NullMask> nullMask;
+        if (nullChunk) {
+            nullMask = nullChunk->getNullMask();
+        }
+        auto minmax_element = [&]<typename T>(std::span<T> data) {
```

Comment:
Can we make this as a function instead of a lambda? The former is more readable to me 😅 .